protocol/communities/community.go

```diff
@@ -1881,6 +1884,9 @@ func (o *Community) VerifyGrantSignature(data []byte) (*protobuf.Grant, error) {
 	if !bytes.Equal(grant.CommunityId, o.ID()) {
 		return nil, ErrInvalidGrant
 	}
+	if grant.Expires < uint64(time.Now().UnixMilli()) {
```

Comment:
I don't think we need milliseconds precision in that case.

Reply:
Only for tests. If only we had a interface for clock, i could be able to mock it and rewind time in tests without actual waiting